backend/inspirehep/indexer/base.py

```diff
@@ -100,10 +100,15 @@ def bulk_index(self, records_uuids, request_timeout=None, max_chunk_bytes=None):
             max_chunk_bytes=max_chunk_bytes,
         )
 
+        failures = []
+        for action_success, action_data in result:
+            if not action_success:
+                failures.append(action_data["index"])
+
         return {
-            "success": success,
+            "success_count": len(records_uuids) - len(failures),
+            "failures_count": len(failures),
             "failures": failures,
-            "failures_count": len(records_uuids) - success,
```

Comment:
tiny: we can store `len(failures)` in a var and not calculate it twice

Reply:
yes, here you are